### PR TITLE
Create 2023-10-24-september-2023.md

### PR DESCRIPTION
**Pull request by arantxarecio**
## PR Checklist
- [x ] Test [Cloud.gov Pages preview](https://pages.cloud.gov/sites/764/builds) for your branch for functionality - make sure all links and buttons work, javascript works as intended, and updates display as expected
  - Preview link: 
- [ x] Conduct a [HemingwayApp](https://hemingwayapp.com/) review of the content for plain language. Request plain language review in #helpwanted if needed before merging this PR
- [ x] Conduct a Lighthouse Audit via Chrome Developer Tools. Identify any issues that must be addressed before merging, and fix
- [ x] Ensure no non-public information is published in the commit messages

### CODE REVIEW

**Comment by arantxarecio** [comment]
Preview: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.sites.pages.cloud.gov/preview/gsa/search-gov-website/arantxarecio-patch-1/about/updates/newsletters/september-2023.html

File: _newsletters/2023-10-24-september-2023.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+### Search system will remain operational
+In the event of a temporary lapse in appropriations, please note that Search.gov will remain in normal operations and maintenance, including the search engine, hosted search results pages, and the Admin Center. API-driven search results pages on your websites are subject to your agency’s guidance.
+
+
+
+## Interested in faceted search?
+
+### Let’s talk about metadata!
+
+As previously announced, our faceted search capability is now available for our customers using API results, and it will soon become available to those customers using hosted results. If you are interested in using this feature for your website, email us at [search@gsa.gov](search@gsa.gov) to schedule a call to talk about getting your metadata ready. The sooner you can start including metadata in your page templates, the sooner we can gather it. We will start using it in results matching right away, and your searchers will be able to use it to filter when the interface is ready.
```
Comment (dawnpm) [inline _newsletters/2023-10-24-september-2023.md:27]: the email link needs to be 
[search@gsa.gov](mailto:search@gsa.gov)

Can you update in both places, here and below, please?

**Review by dawnpm — CHANGES_REQUESTED** [review]
Looks pretty good, one functional change we need to make in two places. thanks.

File: _newsletters/2023-10-24-september-2023.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+### Search system will remain operational
+In the event of a temporary lapse in appropriations, please note that Search.gov will remain in normal operations and maintenance, including the search engine, hosted search results pages, and the Admin Center. API-driven search results pages on your websites are subject to your agency’s guidance.
+
+
+
+## Interested in faceted search?
+
+### Let’s talk about metadata!
+
+As previously announced, our faceted search capability is now available for our customers using API results, and it will soon become available to those customers using hosted results. If you are interested in using this feature for your website, email us at [search@gsa.gov](search@gsa.gov)to schedule a call to talk about getting your metadata ready. The sooner you can start including metadata in your page templates, the sooner we can gather it. We will start using it in results matching right away, and your searchers will be able to use it to filter when the interface is ready.
```
Comment (dawnpm) [inline _newsletters/2023-10-24-september-2023.md:27]: `[search@gsa.gov](mailto:search@gsa.gov)`

**Review by dawnpm — APPROVED** [review]

**Pull request merged**